easybuild/easyconfigs/p/purge_dups/purge_dups-1.2.6-foss-2023b.eb

```diff
@@ -0,0 +1,85 @@
+easyblock = 'PythonBundle'
+
+name = 'purge_dups'
+version = '1.2.6'
+
+homepage = 'https://github.com/dfguan/purge_dups'
+description = "purge haplotigs and overlaps in an assembly based on read depth"
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+builddependencies = [
+    ('pybind11', '2.11.1'),
+]
+
+dependencies = [
+    ('zlib', '1.2.13'),
+    ('bzip2', '1.0.8'),
+    ('Python', '3.11.5'),
+    ('matplotlib', '3.8.2'),
+    ('minimap2', '2.28'),
+]
+
+use_pip = True
+
+default_easyblock = 'MakeCp'
+
+components = [
+    (name, version, {
+        'source_urls': ['https://github.com/dfguan/purge_dups/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}],
+        'checksums': ['75b20df834d28410858ccd4e2d65b6c70849f3cbe5fc05959f38bfc2d041c704'],
+        'start_dir': 'purge_dups-%(version)s',
+        'prebuildopts': "cd src && ",
+        'buildopts': 'CC="$CC" CFLAGS="$CFLAGS"',
+        'files_to_copy': ['bin', 'scripts'],
+    }),
+    ('KMC', '20190629', {
+        'source_urls': ['https://github.com/dfguan/KMC/archive'],
+        'sources': [{'download_filename': 'c05a5f9.tar.gz', 'filename': SOURCE_TAR_GZ}],
+        'patches': [{
+            'name': 'KMC_use_EB_pybind11_and_libs.patch',
```

Comment:
Shouldn't  this include the KMC version (and may some comment or other name like `KMC-dfguan` to indicate that this is for a fork of KMC, as it confused me for a while when I saw your other KMC PR :smile:)?

Reply:
@bedroge Better now?